generated/nidaqmx/_task_modules/channels/ao_channel.py

```diff
@@ -127,7 +128,7 @@ def ao_current_units(self):
         check_for_error(error_code)
 
     @property
-    def ao_custom_scale(self):
+    def ao_custom_scale_name(self):
```

Comment:
The old name is correct, this returns a `Scale` object, not the scale name. The same is true for AI. I know we've talked about this before. So follow whatever we did there.

Reply:
Sure. Updated.